database/context.go

```diff
@@ -35,3 +38,19 @@ func FromContext(c context.Context) Interface {
 func ToContext(c Setter, d Interface) {
 	c.Set(key, d)
 }
+
+// FromCLIContext creates and returns a database engine from the urfave/cli context.
+func FromCLIContext(c *cli.Context) (Interface, error) {
```

Comment:
This logic was copied from the `setupDatabase()` function in the `cmd/vela-server/database.go` file.